include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -159,6 +162,27 @@ indexstoredb_error_get_description(_Nonnull indexstoredb_error_t);
 INDEXSTOREDB_PUBLIC void
 indexstoredb_error_dispose(_Nullable indexstoredb_error_t);
 
+INDEXSTOREDB_PUBLIC bool
+indexstoredb_for_each_symbol_name(_Nonnull indexstoredb_index_t index, _Nonnull indexstoredb_symbol_name_receiver);
+
+INDEXSTOREDB_PUBLIC bool
+indexstoredb_for_each_canonical_symbol_occurence_containing_pattern(
```

Comment:
To be consistent with the other symbol names, I suggest the following:
* `indexstoredb_index_symbol_names`
* `indexstoredb_index_canonical_symbol_occurences_containing_pattern`
* `indexstoredb_index_canonical_symbol_occurences_by_name`

We've been using "index_" in the name since these are effectively methods of the index object, and I've been using plurals instead of "for_each".  I'm not wed to these names, but it would be good to be consistent.